src/simulator/tests/publisher_test.py

```diff
@@ -50,7 +50,7 @@ def test_true_reach_by_spend(self):
 
     def test_liquid_legions_sketch(self):
         sketch = self.publisher.liquid_legions_sketch(0.04)
-        estimator = StandardizedHistogramEstimator(epsilon=np.Inf)
+        estimator = StandardizedHistogramEstimator()
```

Comment:
same as in #16 , getting test failures here with the change

Reply:
See previous comment.